docs/databases/mongodb.md

```diff
@@ -25,7 +25,7 @@ yarn add @graphback/runtime-mongo mongodb
 
 Running either of the two commands will install the `Graphback MongoDB Provider` ([`@graphback/runtime-mongo`](https://www.npmjs.com/package/@graphback/runtime-mongo)) and the MongoDB Driver ([`mongodb`](https://www.npmjs.com/package/mongodb)).
 
-# Creating the Database Connection
```

Comment:
Hmm I have been reserving H1 titles "#" for the main page title and always start with "##".

I just checked and all the titles with # are the same size as the main page title.

I recommend starting with ##

Reply:
Good catch.

I'll change them throughout.

Reply:
Done for this PR and also other changes https://github.com/aerogear/graphback/pull/1700 to some of the docs which I did earlier.